Codeinwp/neve
# fix: allow user agent default styling for <mark> tag

### Summary


### Will affect visual aspect of the product

NO

### Test instructions


- Insert `<mark>` tag or use otter to add a highlight to any text block. It should work.


Closes Codeinwp/neve-pro-addon#934.
